types/rascal/index.d.ts

```diff
@@ -55,6 +55,11 @@ export interface ConnectionAttributes {
     } | undefined;
     socketOptions?: {
         timeout?: number | undefined;
+        service_name: string | undefined;
+        service_description: string | undefined;
+        clientProperties: {
```

Comment:
Are these properties optional? In that case, don't these properties need question marks in front o them? Otherwise, I think, it would not be allowed to omit these properties when specifying them in options.


I'm sorry, I forgot to review sooner.